packages/vue/src/utils/versions.ts

```diff
@@ -0,0 +1,15 @@
+export const nxVersion = require('../../package.json').version;
+export const vueVersion = '^3.3.4';
+export const vueTscVersion = '^1.8.8';
+export const vueRouterVersion = '^4.2.4';
+export const vueTestUtilsVersion = '^2.4.1';
+export const vueEslintConfigPrettierVersion = '^8.0.0';
+export const vueEslintConfigTypescriptVersion = '^11.0.3';
+export const vueTsconfigVersion = '^0.4.0';
+export const tsconfigNode18Version = '^18.2.0';
+export const eslintVersion = '^8.46.0';
+export const eslintPluginVueVersion = '^9.16.1';
+export const vitePluginVueVersion = '^4.3.1';
+export const vueJest3Version = '^29.2.6';
+export const sassVersion = '1.62.1';
+export const lessVersion = '3.12.2';
```

Comment:
Can we group these? Maybe
* vue core
* build deps
* test deps
* linting deps